### DISCUSSION TITLE
SplitCHCRes

### DISCUSSION THREAD

**Discussion by rnpratoori** [idaholab/moose#28197] [Q&A Modules: Phase field]
Hello team,

I am looking at the source file of the kernel `SplitCHCRes` and want to know what are the equations based on which it was modeled. I could not find any documentation for it. Can anyone give me these equations?

**Comment by sourabhkadambi** [comment]
Have a look at this documentation page: https://mooseframework.inl.gov/modules/phase_field/Phase_Field_Equations.html

**Reply by rnpratoori** [reply]
@sourabhkadambi I did have a look at it, but it only shows for `SplitCHWRes` and not for `SplitCHCRes`.
